src/pages/open source for non-developers.md

```diff
@@ -0,0 +1,23 @@
+# Open Source for Non-Developers
```

Comment:
[Praise] I really like the headings! The flow of what -> why -> more-whys is a really good match here IMO. Yay! 🎉

Reply:
Thank you!